hooks/useVoteRecords.ts

```diff
@@ -163,5 +197,71 @@ export default function useVoteRecords(proposal?: ProgramAccount<Proposal>) {
   ])
   ///////
 
+  // for nft-voter
+  useEffect(() => {
```

Comment:
use useAsync or useQuery, not useEffect

Reply:
also i know this code is in the hell zone and you are just copying patterns, but more comments would be nice just explaining whats going on (what's this data? why are we getting it?) because its not the easiest to follow if you are low brainpower like me